src/components/feedback-form.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+// import Form from "./form";
+import styles from "./call-to-action.module.css";
+import PropTypes from "prop-types";
```

Comment:
These two inputs can be removed as well because the imports are not being used.

Reply:
I added an issue to the repo to improve the developer experience. The ESLint configuration should be more helpful here.

https://github.com/fullstacknights/website/issues/45